frontend/src/app/admin/premium-domains/page.tsx

```
@@ -0,0 +1,1204 @@
+                            >
+                                <div className="flex items-center justify-between">
+                                    <div className="flex-1">
+                                        <div className="flex items-center space-x-3">
+                                            <div className="flex-shrink-0">
+                                                {expandedDomains.has(domain.id) ? (
+                                                    <svg
+                                                        className="w-5 h-5 text-yellow-500 dark:text-yellow-400 transition-transform duration-200"
+                                                        fill="none"
+                                                        stroke="currentColor"
+                                                        viewBox="0 0 24 24"
+                                                    >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)